mixer/pkg/attribute/protoBag.go

```diff
@@ -65,7 +65,7 @@ func (pb *ProtoBag) Get(name string) (interface{}, bool) {
 	// find the dictionary index for the given string
 	index, ok := pb.getIndex(name)
 	if !ok {
-		glog.Warningf("Attribute '%s' not in either global or message dictionaries", name)
+		glog.V(4).Infof("Attribute '%s' not in either global or message dictionaries", name)
```

Comment:
This indicates a real problem, I think Warningf is warranted here.

Reply:
Oh, I take it back.

It's OK for this message to show up, it just means a config is referencing an attribute that wasn't sent by the proxy or generated by an APA.